### ISSUE TITLE
E001 false positive from echo lines containing PowerShell if ( syntax

### ISSUE THREAD

**Issue by tboy1337** [tboy1337/Blinter#34]
## Summary

Blinter reports **E001** (unclosed parenthesis blocks) when batch files use `echo` to write PowerShell helper scripts whose content contains `if (` tokens, even though the batch `IF/FOR` structure is balanced.

## Reproduction

A batch subroutine that writes a `.ps1` file via parenthesized `echo` blocks containing PowerShell such as:

```bat
:write_helper
(
echo try {
echo     if ($path -notlike "*$binPath*") {
echo         Write-Host 'done'
echo     }
echo     exit 0
echo }
echo catch {
echo     Write-Host "ERROR: $_"
echo     exit 1
echo }
) > "%TEMP%\helper.ps1"
exit /b 0
```

## Actual

- **E001** near the subroutine (for example: `3 unclosed parenthesis block(s) in IF/FOR structure`)

## Expected

No E001. The `if (` substrings are literal PowerShell text inside `echo` lines, not batch `IF` headers.

## Notes

The global parenthesis depth tracker in `exit_flow.py` does not distinguish echoed script content from batch block structure.

## Environment

- Blinter 1.1.6
- Observed while linting `scripts/install_blinter.cmd`

**Comment by tboy1337** [comment]
Fixed in https://github.com/tboy1337/Blinter/commit/8f32abaf1100af6b07b1948b205395618a12e8b3
